server/src/main/java/org/elasticsearch/persistent/PersistentTasksCustomMetaData.java

```diff
@@ -189,7 +190,7 @@ public long getNumberOfTasksOnNode(String nodeId, String taskName) {
 
     @Override
     public Version getMinimalSupportedVersion() {
-        return Version.V_5_4_0;
+        return Version.V_6_3_0;
```

Comment:
I'm not sure we can change this - 6.2 with xpack will have problems.

Reply:
I pushed a18f16628f6045dd57e0b3e5dce790b8afd623f4.